examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3717,6 +3717,187 @@ provisional cluster ElectricalEnergyMeasurement = 145 {
   readonly attribute int16u clusterRevision = 65533;
 }
 
+/** Electric Vehicle Supply Equipment (EVSE) is equipment used to charge an Electric Vehicle (EV) or Plug-In Hybrid Electric Vehicle. This cluster provides an interface to the functionality of Electric Vehicle Supply Equipment (EVSE) management. */
+provisional cluster EnergyEvse = 153 {
+  revision 1; // NOTE: Default/not specifically set
```

Comment:
In my copy of the spec I have "Energy EVSE Cluster" at revision 2 with note "updates after 0.7 ballot review"

Reply:
How is this meant to be set - I think in the zap file I have it set to 2... but the .matter file seems to set it to 1?

Reply:
set it in the XML side instead. ZAP seems to be forcing endpoint config.

Reply:
like here for management: https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml#L85

Reply:
Concern is that otherwise any other app will pick up "version 1" by default and zap has to be always changed. That is error prone (beside our rest of the tooling seemingly not liking it)

Reply:
Bug seems to be in the XML. Use `<globalAttribute side="either" code="0xFFFD" value="2" />` (currently it says `side="server"` and that is insufficient)

Reply:
Per slack. Looks like you have raised a new issue to track this across multiple clusters.
https://github.com/project-chip/connectedhomeip/pull/31009

Thanks for finding that.